stickermanager.py

```diff
@@ -251,6 +252,13 @@ async def _locked_start_poll(event: Union[events.NewMessage.Event, Message]) ->
         if not orig_evt.photo and (not orig_evt.sticker or
                                    orig_evt.sticker.mime_type == 'application/x-tgsticker'):
             return
+        if emoji is None and orig_evt.sticker:
+            for attr in orig_evt.sticker.attributes:
+                if isinstance(attr, DocumentAttributeSticker):
+                    emoji = attr.alt
+                    break
+        if emoji is None:
+            return
```

Comment:
[Read the docs!](https://docs.telethon.dev/en/latest/quick-references/objects-reference.html#file)

```suggestion
        if not orig_evt.file.emoji:
            break
```

Reply:
I don't know why I forgot about that. :laughing: